src/modules/extendedMap/i18n/en_AU.root.json

```diff
@@ -26,6 +26,28 @@
             "static": "static",
             "title": "Center map: Mode"
         },
+        "mapStyleFilter": {
+            "description": "Aktiviert ein paar kosmetische Filter für die Karte",
```

Comment:
German in english Land?

Reply:
VwVfG §23 Abs. 1
Die Amtssprache ist deutsch.
Source: https://www.gesetze-im-internet.de/vwvfg/__23.html 
😛😂

Gonna fix it in a minute